autoload/beancount.vim

```diff
@@ -56,14 +66,29 @@ function! beancount#complete_account(findstart, base)
         endif
         let b:beancount_accounts = beancount#find_accounts(l:root)
     endif
-    let l:pattern = '\V' . substitute(a:base, ":", '\\S\\*:\\S\\*', "g")
+
+    if g:beancount_account_completion ==? 'chunks'
+        let l:pattern = '\V' . substitute(a:base, ":", '\\[^ \\t:]\\*:', "g") . '\[^ \t:]\*'
```

Comment:
Yes, `g:beancount_detailed_first` is more or less a leftover from some changes I cherry-picked from another branch. Maybe I should remove that from this pull request to keep things separate.